src/fusor/harvester.py

```diff
@@ -0,0 +1,166 @@
+"""Harvester methods for output from different fusion callers"""
+
+import csv
+from abc import ABC
+from pathlib import Path
+from typing import ClassVar
+
+from fusor.fusion_caller_models import (
+    JAFFA,
+    Arriba,
+    Cicero,
+    EnFusion,
+    FusionCaller,
+    FusionCatcher,
+    Genie,
+    STARFusion,
+)
+
+
+class FusionCallerHarvester(ABC):
+    """ABC for fusion caller harvesters"""
+
+    fusion_caller: FusionCaller
+    column_rename: dict
+    delimeter: str
+
+    def load_records(
+        self,
+        fusion_path: Path,
+    ) -> list[FusionCaller]:
+        """Convert rows of fusion caller output to Pydantic classes
+
+        :param path: The path to the fusions file
+        :param column_rename: A dictionary of column mappings
+        :param delimeter: The delimeter for the fusions file
+        :raise ValueError: if the file does not exist at the specified path
+        :return: A list of fusions, represented as Pydantic objects
+        """
+        if not fusion_path.exists():
+            statement = f"{fusion_path!s} does not exist"
+            raise ValueError(statement)
+        fusions_list = []
+        fields_to_keep = self.fusion_caller.__annotations__.keys()
```

Comment:
Wondering if `self.fusion_caller.__annotations__` is a dict, if so we can probably change to

```suggestion
        fields_to_keep = self.fusion_caller.__annotations__
```